app/models/compute_resource.rb

```diff
@@ -6,6 +6,11 @@ class ComputeResource < ActiveRecord::Base
   include Parameterizable::ByIdName
   encrypts :password
 
+  attr_accessible :name, :provider, :description, :url, :display_type,
+      :set_console_password, :user, :password, :public_key,
+      :server, :tenant, :project, :email, :key_path,
+      :location_ids, :organization_ids
```

Comment:
Ditto in most models, all `_ids` attributes should have a corresponding `_names` listed and vice versa.